src/lib/api.ts

```diff
@@ -586,15 +586,15 @@ export class API {
 		module: object,
 		fallback?: Record<never, never>,
 	): Promise<Record<string, unknown>> {
-		const output: Record<string, unknown> = {
-			...fallback,
-			...(typeof out === "object" ? out : undefined),
-		};
+		const output: Record<string, unknown> | unknown[] = Array.isArray(out)
```

Comment:
Maybe the type of `fallback` should be changed to accept other types, or it will be totally useless for array inputs and not aligned with objects.

> But actually, I am wondering if `fallback` is needed here, since I haven't used it myself. 🤔

Reply:
I admit ignored fallback as in the only call to parse_output it is not set.